pkg/config/config.go

```diff
@@ -251,8 +255,12 @@ type Config struct {
 	// not enabled. Defaults to "http".
 	DefaultExternalScheme string
 
-	// DefaultExternal specifies whether internal traffic is encrypted or not.
+	// InternalEncryption specifies whether internal traffic is encrypted or not.
 	InternalEncryption bool
+
+	// InternalTrust specifies what level of trust whether internal traffic is encrypted with mutual trust between peers.
+	// InternalTrust enforces InternalEncryption = true
```

Comment:
Do we want still want to force InternalEncryption to "true", or do we want this to define additional settings if InternalEncryption is "true"?

Reply:
missd this - will fix

Reply:
Removed.